### DISCUSSION TITLE
Adjust the Kat hitfinish to a more standard hitsound for the `osu! "argon" pro`'s osu!taiko skin

### DISCUSSION THREAD

**Discussion by aceticke** [ppy/osu#24262] [Ideas]
Tile is self explanatory, I like the new hitsounds but I feel as if the Japanese vocalisation should be kept to the skin aimed at a more casual audience.

**Comment by bdach** [comment]
related/duplicate: https://github.com/ppy/osu/discussions/24239

**Reply by aceticke** [reply]
Related yes, duplicate I wouldn't say so (I read this discussion before posting), would rather this just be bundled as a default rather than me having to delete it myself/make a new hitsound for it
